frontend/src/eventPage/Overview.js

```diff
@@ -6,8 +6,8 @@ import { displayName, formattedAmount } from '../helperFunctions/helperFunctions
 
 export default function Overview() {
   const { events } = useContext(EventContext);
-  const { id } = useParams();
-  const event = events.find((event) => event.id === id);
+  const { eventId } = useParams();
+  const event = events.find((event) => event.id === eventId);
 
   return (
   event ? event.members.map((member) => (
```

Comment:
Only <li> are allowed as children of <ul> an <ol>

Reply:
??

Reply:
Sry, only `<li>` are allowed as children of lists